OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -101,13 +101,36 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Altitude above terrain below which to explode. Zero effectively deactivates airburst.")]
 		public readonly WDist AirburstAltitude = WDist.Zero;
 
+		[Desc("Length of the contrail (in ticks).")]
```

Comment:
```suggestion
		[Desc("When set display a line behind the actor that fades out in the end.")]
```

Reply:
Changed it to `When set, display a line behind the actor. Length is measured in ticks after appearing.`

Reply:
Length here means "duration" and not "distance" even though that's probably how people would think about this. I guess that description is OK :shrug: